kong/global.lua

```diff
@@ -162,7 +164,7 @@ do
       worker_events     = worker_events,
       propagation_delay = kong_config.db_update_propagation,
       ttl               = db_cache_ttl,
-      neg_ttl           = db_cache_ttl,
+      neg_ttl           = db_cache_neg_ttl,
```

Comment:
```suggestion
      neg_ttl           = db_cache_neg_ttl or db_cache_ttl,
```

This would keep backwards compatibility by allowing the user to set both at once via `db_cache_ttl`, like they can now.